src/smc-webapp/project_files.cjsx

```diff
@@ -1461,10 +1490,69 @@ ProjectFilesSearch = rclass
         selected_file_index : 0
         num_files_displayed : 0
 
+    getInitialState : ->  # Miniterm functionality
+        stdout : undefined
+        state  : 'edit'   # 'edit' --> 'run' --> 'edit'
+        error  : undefined
+
+    # Miniterm functionality
```

Comment:
I'm confused - why is this big block of coded added, but no corresponding deleting?  Did you just copy the same code again.... seems ugly.